src/main/java/org/jabref/model/search/query/SearchQuery.java

```diff
@@ -148,14 +124,12 @@ public EnumSet<SearchFlags> getSearchFlags() {
      */
     public List<String> getSearchWords() {
         if (searchFlags.contains(SearchFlags.REGULAR_EXPRESSION)) {
-            return Collections.singletonList(query);
+            return Collections.singletonList(searchExpression);
         }
         if (!isValid()) {
             return List.of();
         }
-        return Arrays.stream(QueryTermExtractor.getTerms(parsedQuery))
-                     .map(WeightedTerm::getTerm)
-                     .toList();
+        return List.of();
```

Comment:
Is the whole method needed? If it always returns the empty list, it can be removed?